.github/workflows/check-catalyst.yaml

```diff
@@ -341,22 +343,23 @@ jobs:
       - name: Build Runtime test suite for Lightning simulator
         if: ${{ matrix.simulator == 'lightning' }}
         run: |
-            cmake -S runtime -B build -G Ninja -DCMAKE_BUILD_TYPE=Debug \
-                                      -DENABLE_LIGHTNING_KOKKOS=OFF \
-                                      -DLIGHTNING_GIT_TAG=$LIGHTNING_GIT_TAG_VALUE \
-                                      -DQIR_STDLIB_LIB="$(pwd)/runtime/qir-stdlib-build/" \
-                                      -DQIR_STDLIB_INCLUDES="$(pwd)/runtime/qir-stdlib-build/include/" \
-                                      -DENABLE_CODE_COVERAGE=ON
+            cmake -S runtime -B build -G Ninja \
+                  -DCMAKE_BUILD_TYPE=Debug \
+                  -DENABLE_LIGHTNING_KOKKOS=OFF \
+                  -DLIGHTNING_GIT_TAG=$LIGHTNING_GIT_TAG_VALUE \
+                  -DQIR_STDLIB_LIB="$(pwd)/runtime/qir-stdlib-build/" \
+                  -DQIR_STDLIB_INCLUDES="$(pwd)/runtime/qir-stdlib-build/include/" \
+                  -DENABLE_CODE_COVERAGE=ON
             cmake --build build --target runner_tests -j$(nproc)
 
       - name: Build Runtime test suite for Lightning-Kokkos simulator
         if: ${{ matrix.simulator == 'lightning-kokkos' }}
         run: |
-            cmake -S runtime -B build -DCMAKE_BUILD_TYPE=Release \
-                                      -DENABLE_LIGHTNING_KOKKOS=ON \
-                                      -DQIR_STDLIB_LIB="$(pwd)/runtime/qir-stdlib-build/" \
-                                      -DQIR_STDLIB_INCLUDES="$(pwd)/runtime/qir-stdlib-build/include/" \
-                                      -G Ninja
+            cmake -S runtime -B build -G Ninja \
+                  -DCMAKE_BUILD_TYPE=Release \
+                  -DENABLE_LIGHTNING_KOKKOS=ON \
```

Comment:
@maliasadi Shouldn't we include the git tag (`-DLIGHTNING_GIT_TAG=$LIGHTNING_GIT_TAG_VALUE \`) here for kokkos as well?

Reply:
The Kokkos support needs to be updated/extended before using the git tag. We can address both when we revisit this support.